### ISSUE TITLE
Add yaml verification to build process

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#2273]
We have a variety of yaml files within the build

For example under .azure-pipelines are yaml files which control the build process.

docker-compose & helm also make use of yaml files

Errors in these are not picked up in a normal dev build or PR build, which can result in breakages only being found when a build pipeline is executed, or a docker compose image is used.

For my own development I use 'yamllint' to verify my files ie
```
➜  common git:(issue353b) ✗ yamllint ci-steps-template.yml
ci-steps-template.yml
  10:3      error    syntax error: expected <block end>, but found '?'
  11:3      error    wrong indentation: expected 4 but found 2  (indentation)
  17:81     error    line too long (87 > 80 characters)  (line-length)
  22:81     error    line too long (87 > 80 characters)  (line-length)
  44:81     error    line too long (135 > 80 characters)  (line-length)
  47:81     error    line too long (87 > 80 characters)  (line-length)
  51:81     error    line too long (87 > 80 characters)  (line-length)

```

The line too longs can be ignored (and often hard to avoid), but the indendation and syntax errors do cause a problem

Opening up this issue to see if we could introduce such a check usign yamllint, or a pluging we can find, or perhaps checkstyle into the build to detect these simple format issues early

**Comment by planetf1** [comment]
Should help prevent issues like that fixed in PR #2274

**Comment by planetf1** [comment]
We now use github actions, and a PR will cause validation of these files.
For helm we do use yaml, but there is much more additional processing required for templating and so simple yaml validation
will result in false positives. 

As such closing
